src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.hpp

```diff
@@ -279,74 +185,48 @@ void spacetime_deriv_damped_harmonic_h(
     double sigma_r) noexcept;
 
 /*!
- * \brief Spacetime derivatives of the damped harmonic gauge source function.
- *
- * \details See spacetime_deriv_damped_harmonic_h() for details.
+ * \brief Compute tag for the damped harmonic gauge source function and its
+ * spacetime derivative.
  *
- * \note As `DampedHarmonicHCompute`, this compute tag also:
- *   1. fixes amplitude factors to  \f$ A_{L1} = A_{L2} = A_{S} = 1 \f$,
- *   2. fixes exponents to \f$ e_{L1} = e_{L2} = e_{S} = 4 \f$,
- *   3. fixes roll-on start time to be equal for all terms in \f$H_a\f$, i.e.
- *     \f$ t_{0,L1} = t_{0,L2} = t_{0,S} \f$,
- *   4. fixes roll-on timescale to be equal for all terms in \f$H_a\f$, i.e.
- *     \f$ \sigma_t^{L1} = \sigma_t^{L2} = \sigma_t^{S} \f$.
- * \note
- * The first two set of parameters are set to reproduce \cite Deppe2018uye, and
- * not through input options. They would need to be changed to reproduce,
- * for instance, the gauge source of \cite Szilagyi2009qz. Values of the last
- * two set of parameters are set through input options.
+ * \see damped_harmonic()
```

Comment:
The `\see` link is broken (needs a namespace I think)

Reply:
Turns out to be a missing cond. I've pushed a fixup.